src/qibosoq/programs/flux.py

```diff
@@ -56,7 +57,7 @@ def set_bias(self, mode: str = "sweetspot"):
                 stdysel="last",
                 freq=0,
                 phase=0,
-                gain=int(max_gain * qubit.bias),
+                gain=int(max_gain * qubit.bias),  # type: ignore
```

Comment:
Here there is an explicit cast, so I'm not imagining why you need to disable the type check...

Reply:
qubit.bias can be None (not here, but in general) so I get an error since I could be multiplying int and None

Reply:
Instead of ignoring the error, if this property is lost by the type you receive (because you generate a type that is not `None`, but then you only declare the broader class) assert that the property still holds.

```py
assert qubit.bias is not None
```

The best would be to be able to do it statically, such that at linting time you're already sure that it won't be `None`, since it's something that you know to be true. However, if you're not able to state statically, at least tell it dynamically (such that sooner or later we would get an error in case of violation, either from `mypy` or from Python itself).

Reply:
some line up I have an if to break the for... I kinda like the idea and it seems already enough
I thought of modifying it like this:
```python
if qubit.bias is None or flux_ch is None or qubit.bias == 0:
                continue
```
what do you think?

Reply:
Or even adding a property like "qubit.fluxed"

Reply:
Both options should be fine, with the second being good if you want to add extra docs explanations.

Reply:
Mypy does not seem to like the property... still same error. I have now resorted to the if